lib/kubernetes-deploy/ejson_secret_provisioner.rb

```diff
@@ -68,6 +68,14 @@ def private_key
       @private_key ||= fetch_private_key_from_secret
     end
 
+    def ejson_keys_secret
+      @ejson_keys_secret ||= begin
+        out, err, st = @kubectl.run("get", "secret", EJSON_KEYS_SECRET, output: "json")
+        out = JSON.parse(out) if st.success?
+        [out, err, st]
+      end
```

Comment:
Since the error handling is slightly different when this is called, I'm compelled to return the `out, err, st` and let the caller sort it out. Said that, calling this method `ejson_keys_secret` might not be accurate, since it encompasses more information than just the secret itself. Maybe `kubectl_get_ejson_keys`? That's more explicit but exposes implementation details 🤷‍♂️